StringCalculator/StringCalculator.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace StringCalculator;
+public class StringCalculator
+{
+    private string[] seperators = new string[] { ",", "\\n" };
+    private List<string> negativeNumbers = new List<string>();
+    private int maxNumberToAdd = 1000;
+    private int finalResult = 0;
+
+    public int Add(string numbersGiven)
+    {
+        string numbersToCalculate = numbersGiven;
```

Comment:
You don't need this here, you may as well do the assignment when you assign numbersToCalculate from CheckForSeparators

Reply:
You only need to be careful of things like this when you are directly **mutating** parameter values.